src/NzbDrone.Mono/ProcMountProvider.cs

```diff
@@ -17,23 +18,26 @@ public class ProcMountProvider : IProcMountProvider
     {
         private static string[] _fixedTypes = new [] { "ext3", "ext2", "ext4", "vfat", "fuseblk", "xfs", "jfs", "msdos", "ntfs", "minix", "hfs", "hfsplus", "qnx4", "ufs", "btrfs" };
         private static string[] _networkDriveTypes = new [] { "cifs", "nfs", "nfs4", "nfsd", "sshfs" };
+        private static readonly Regex OctalRegex = new Regex(@"\\\d{3}", RegexOptions.Compiled);
 
         private static Dictionary<string, bool> _fileSystems;
 
+        private readonly IDiskProvider _diskProvider;
         private readonly Logger _logger;
 
-        public ProcMountProvider(Logger logger)
+        public ProcMountProvider(IDiskProvider diskProvider, Logger logger)
```

Comment:
Can't do that, circular dependency. ProcMountProvider is part of the Mono DiskProvider implementation, basically a composition. That's why it was allowed to directly access System.IO.File and such.